ckanext/syndicate/tasks.py

```diff
@@ -124,8 +143,17 @@ def _create_package(package):
 
     new_package_data['extras'] = filter_extras(new_package_data['extras'])
     new_package_data['resources'] = filter_resources(package['resources'])
-    new_package_data['owner_org'] = get_syndicated_organization()
-    del new_package_data['organization']
+
+    org = new_package_data.pop('organization')
+
+    try:
+        if not is_organization_preserved():
+            raise ValueError("Ignore organization")
```

Comment:
This seems a bit of an odd way of handling the case where the config option isn't set. Why not:
```
if is_organization_preserved():
    org_id = replicate_remote_organization(org)
else:
    org_id = get_syndicated_organization()
```
If we are expecting `replicate_remote_organization()` to throw an exception we should handle that specifically